src/Utils.ts

```diff
@@ -0,0 +1,13 @@
+import Web3 = require('web3');
+
+export class Utils {
+
+  /**
+   * It provides checksum address using web3.
+   * @param address Address.
+   * @returns It returns checksum address.
+   */
+  public static toChecksumAddress(address: string): string {
```

Comment:
We should add unit tests for this method.

Reply:
Added unit test case.